build.gradle

```diff
@@ -150,6 +142,10 @@ project('common') {
         //Do NOT add any additional dependencies here.
     }
 
+    tasks.withType(JavaCompile) {
+        options.compilerArgs.addAll(['--release', '8'])
```

Comment:
can a build property be defined to control what Java version to use for Client and its dependencies? This is for the case if someone wants to checkout the code and build everything using JDK11. Currently there is a need to make edits in multiple places in `biuild.gradle`. It would be a much better experience if there will be a single property that controls it.

Reply:
I looked into that, but couldn't find any sort of function/option that can programmatically apply some config to dependent tasks. The best we can do is have some sort of flag and a check in each of these `task.withType(JavaCompile)` blocks.

Reply:
That should work. As long as there is one place to turn Java11 on for Client and its dependencies - it's good enough